### ISSUE TITLE
Color contrast ratio of focus indicator over 'toolStripSplitButton1' is 1.247:1 which is less than 3:1: A11y_.NET CoreWinforms_StripControls_Toolstripwithmenuitem_NonTextContrast

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/winforms#11450]
### .NET version

dotnet-sdk-9.0.100-preview.5.24258.1

### Did it work in .NET Framework?

Yes

### Issue description

GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #BM_.NET Core[Winforms]_Win32_May2024; #.NETCore; #WCAG1.4.11; #DesktopApp; #Win11; #FTP; #NonTextContrast;

### Environment Details:
App name: .NET Core Winforms
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "MenuToolbars_Controls:Testing the controls under Menu Toolbars Tab" and hit ENTER key.
4. TAB to StripControls button and hit ENTER key.
5. TAB to 'toolStripSplitButton1' present at below section of the screen.
6. Observe the issue.

### Actual Result:
Color contrast ratio of focus indicator over 'toolStripSplitButton1' is 1.247:1 which is less than 3:1.

**Similar issue is also observed in below scenario:
Issue 1: ToolStripContainer**

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "MenuToolbars_Controls:Testing the controls under Menu Toolbars Tab" and hit ENTER key.
4. TAB to ToolStripContainer button and hit ENTER key.
5. TAB to 'toolStripSplitButton2' present at below section of the screen.
6. Observe that color contrast ratio over "toolStripSplitButton2" is failing.

**Issue2: PrintingTesting_PrintPreview:**

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "Printing_Controls:Testing the controls under Printing Tab" and hit ENTER key.
4. TAB to Preview button and hit ENTER key.
5. Print Preview Screen would appear.
6. Navigate to "Zoom" button using TAB key and observe that color contrast ratio over "Zoom" is failing.

### Expected Result:
Color contrast ratio of focus indicator over 'toolStripSplitButton1' should be greater than or equal to 3:1

### User Impact:
Low vision users will not be able to identify where the focus is and will not be able to interact with the control.

### Attachment
![11450_A11y_ NET CoreWinforms_StripControls_Toolstripwithmenuitem_NonTextContrast](https://github.com/dotnet/winforms/assets/90901519/ba224b99-6c31-4e00-8e58-7469bddfc22b)

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by merriemcgaw** [comment]
@LeafShi1 can you change the border of all selected ToolStripItems to a solid dark line? Make sure the contrast works in the high contrast themes as well.

**Comment by LeafShi1** [comment]
@Olina-Zhang Can your team help test whether other controls have the same problem?

**Comment by Olina-Zhang** [comment]
> @Olina-Zhang Can your team help test whether other controls have the same problem?

Other controls haven't this problem.

**Comment by merriemcgaw** [comment]
@Epica3055 @Tanya-Solyanik is this on track for RC1?

**Comment by PoojaNamde** [comment]
@Epica3055 Could you please upload latest App zip file to verify this issue.

**Comment by MandiMan** [comment]
Verified this issue in the latest .NET 9.0.100-rc.1.24381.10, it was fixed: the color contrast ratio of focus indicator over 'toolStripSplitButton1' is greater than 3:1.
![image](https://github.com/user-attachments/assets/d2567256-503e-4138-b382-1455b23a3e87)
In addition, the default value of the RenderMode property of StatusStrip is now changed, 9.0 is changed to ManagerRenderMode by default, 8.0 is System.
![image](https://github.com/user-attachments/assets/ae9f3f1c-c46f-4160-8d0a-7851ea5b2a95)

**Comment by PoojaNamde** [comment]
@MandiMan Similar issues provided in repro steps are still reproducing. Please resolve these issues and reactivate this bug.

**Comment by MandiMan** [comment]
@Epica3055 @PoojaNamde verified this issue on the main branch of the Winforms repo that **Issue1(ToolStripContainer)** of similar issues has been fixed, the color contrast ratio of focus indicator on 'toolStripSplitButton2' is greater than 3:1.
![image](https://github.com/user-attachments/assets/5f0570f3-5637-42e7-b05c-80d53be635c4)
**Issue2(PrintingTesting_PrintPreview)** of similar issues still repro, the color contrast ratio of focus indicator on 'Zoom' is less than 3:1.
![image](https://github.com/user-attachments/assets/7911a9c5-e97a-4277-bd2a-438e1c1fcf69)

**Comment by MelonWang1** [comment]
Verified the issue with .NET 9.0.100-rc.1.24422.10 test pass build that the issue has been fixed, which have the same results as above.

**Comment by PoojaNamde** [comment]
@MelonWang1 could you please share .NET 9.0.100-rc.1.24422.10 test pass build details to verify this issue.

**Comment by PoojaNamde** [comment]
@MelonWang1 Verified both the issue in latest environment but issue is still reproducing.

![Screenshot (1174)](https://github.com/user-attachments/assets/3211afc9-63e8-40f6-bebc-875eab590f5b)

![Screenshot (1175)](https://github.com/user-attachments/assets/d07d71bf-048c-4869-aecd-11021ab7299d)

**Comment by MelonWang1** [comment]
@PoojaNamde Please verify again using new upload file.

**Comment by MelonWang1** [comment]
@Epica3055 Reopened since issue 2 was not fixed.

**Comment by PoojaNamde** [comment]
@Epica3055 Could you please upload the latest build so that I can reverify the issue in latest build.

**Comment by LeafShi1** [comment]
> @Epica3055 Could you please upload the latest build so that I can reverify the issue in latest build.

I uploaded the private dll to [DotNet - WinForms - All Documents (sharepoint.com)](https://microsoft.sharepoint.com/teams/DotNetTeam/Shared%20Documents/Forms/AllItems.aspx?id=%2Fteams%2FDotNetTeam%2FShared%20Documents%2FAccessibility%2FTest%20Pass%20Instructions%2F%2ENET%209%20Test%20Pass%20Scenarios%2FWinForms&viewid=9dc270e3%2D8c2b%2D45b2%2Dbd30%2Dd2502ae0249c), please use `PrivateBuildForTest` to test

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:09-20-24;

Verified the bug in environment
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment

![image](https://github.com/user-attachments/assets/b5234be4-c59f-4929-b568-ba2f5c150bda)

**Comment by Tanya-Solyanik** [comment]
breaking change issue - https://github.com/dotnet/docs/issues/43957
